packages/components/pie-modal/src/modal.scss

```diff
@@ -29,7 +29,7 @@
 
     // We need to override the icon sizes at different screen sizes regardless of size prop passed in
     pie-icon-button {
-        @media (max-width: calc($breakpoint-wide - 1px)) {
+        @media (width <= calc($breakpoint-wide - 1px)) {
```

Comment:
this was automatically suggested / fixed by `stylelint`

more details are found [here](https://stylelint.io/user-guide/rules/media-feature-range-notation/)

Reply:
Can it be just `width < $breakpoint-wide`?

Reply:
I've tried removing the calc as suggested but it doesn't seem to work. am I missing something?

https://www.w3schools.com/cssref/tryit.php?filename=trycss3_media_bg